### PR TITLE
speed up build time

### PR DESCRIPTION
**Pull request by maddygriz**

### CODE REVIEW

File: .BBSoptions [hunk 1]
```diff
@@ -0,0 +1 @@
+RunLongTests: TRUE
```
Comment (NicoleEO) [inline .BBSoptions:1]: Add another line

File: tests/testthat/test_all.R [hunk 2]
```diff
@@ -0,0 +1,1245 @@
+tifFile <- downloadMouseBrainImage()
+
+annots <- system.file("extdata", "muBrain_LabWorksheet.txt", 
+                      package = "SpatialOmicsOverlay")
+
+annotsDF <- readLabWorksheet(lw = annots, slideName = "4")
+
+##################################  Utils  #####################################
+testthat::test_that("mouse brain tiff can be downloaded",{
```
Comment (NicoleEO) [inline tests/testthat/test_all.R:9]: I'm guessing specs will be updated later as this is just a doc update.

**Review by NicoleEO — COMMENTED** [review]
Good except the options file return

File: tests/testthat/test_all.R (hunk 2 above)
Comment (maddygriz) [reply tests/testthat/test_all.R:9]: Yep I can update specs tomorrow. some of the specs will be tested in the longtests, fyi

**Pull request merged**